include/IECore/MeshAlgo.h

```diff
@@ -47,6 +47,23 @@ namespace IECore
 namespace MeshAlgo
 {
 
+class TexturePrimVarNames
+{
+	public:
+		TexturePrimVarNames( const std::string &positionPrimVarName, const std::string &uvSetName );
+
+		std::string positionPrimVarName;
+		std::string uvSetName;
+
+		std::string uName() const;
+		std::string vName() const;
+
+		std::string tangentName() const;
```

Comment:
Like I said on the Gaffer PR, I'm not sure I'm convinced by this direction of fixed names, but if we do want them, then rather than have an extra utility class, perhaps we should just refactor so that we have free functions to add/retrieve the tangents for a specific mesh/uvSet combo?